bundles/org.openhab.core.io.transport.mqtt/src/main/java/org/openhab/core/io/transport/mqtt/MqttBrokerConnection.java

```diff
@@ -575,13 +569,14 @@ public boolean hasSubscribers() {
      * @param topic The topic to subscribe to.
      * @return Completes with true if successful. Exceptionally otherwise.
      */
-    protected CompletableFuture<Boolean> subscribeRaw(String topic) {
+    protected CompletableFuture<Boolean> subscribeRaw(String topic, Subscription subscription) {
         logger.trace("subscribeRaw message consumer for topic '{}' from broker '{}'", topic, host);
         CompletableFuture<Boolean> future = new CompletableFuture<>();
         final MqttAsyncClientWrapper mqttClient = this.client;
         if (mqttClient != null && mqttClient.getState().isConnected()) {
-            mqttClient.subscribe(topic, qos, clientCallback).whenComplete((s, t) -> {
+            mqttClient.subscribe(topic, qos, subscription).whenComplete((s, t) -> {
                 if (t == null) {
+                    logger.trace("Successfully subscribed to topic {}", topic);
                     future.complete(true);
                 } else {
                     logger.warn("Failed subscribing to topic {}", topic, t);
```

Comment:
This line of code has not been touched by it looks like there is a placeholder missing in the logger message or we are passing an argument too much.

Reply:
Actually no. If the last argument to any slf4j logger is a Throwable, then the throwable will be logged even if there is not placeholder for the it.

Reply:
Thanks for clarification. I was not aware that `t` is a `Throwable`.